warp/tests/test_static.py

```
@@ -346,6 +346,59 @@ def static_loop_variable(results: wp.array(dtype=int)):
+            counts = count_ssa_occurrences(static_loop_force_unroll, ["add", "for"])
+            test.assertEqual(counts["add"], 8, "Static for loop must be unrolled")
+            test.assertGreaterEqual(counts["for"], 1, "Static for loop must be unrolled")
+            test.assertLessEqual(counts["for"], 2, "Static for loop must be unrolled")
+    finally:
+        wp.set_module_options(original_options)
+
+
+def test_static_for_loop_no_force_unroll(test, device):
+    @wp.kernel
+    def static_loop_no_force(results: wp.array(dtype=int)):
+        s = int(0)
```

Ruff (0.14.4)

Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)